runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -1026,22 +1032,32 @@ TR::J9S390InterfaceCallDataSnippet::emitSnippetBody()
    getSnippetLabel()->setCodeLocation(cursor);
    TR::Node * callNode = getNode();
 
+   // J2I relocation information for private nestmate calls
+   auto j2iRelocInfo = reinterpret_cast<TR_RelocationRecordInformation*>(comp->trMemory()->allocateMemory(sizeof(TR_RelocationRecordInformation), heapAlloc));
```

Comment:
All similar points to above need to be addressed for this one as well.

Reply:
Variable scope reduced.